### DISCUSSION TITLE
My Student ID Expire Date Is extended for Global COVID Situation May I extend the program Duration?

### DISCUSSION THREAD

**Discussion by geniusmonir** [community/community#38258] [GitHub Education]
Hi There,
Recently My Student ID Card expire date is extended that is already ended in GitHub Student Pack. Can I extend this now?

**Comment by samus-aran** [comment]
Hi @geniusmonir 👋 

How did you get on? We accept other forms of evidence to verify you. Do you have another dated document that can prove the extension of the academic study?

**Reply by geniusmonir** [reply]
Yes I have student ID card,

New ID Card,
[Edited] 
Old ID Card (Scanned and used to verify first time)
[Edited]

**Reply by samus-aran** [reply]
Well done @geniusmonir 🎉  Glad you got verified!

I've removed your private information from [this post](https://github.com/orgs/community/discussions/38258#discussioncomment-4228923) as posting identifiable information is not allowed, per our [Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md). 🙂
